### PR TITLE
Extend Prometheus kernel stats

### PR DESCRIPTION
**Pull request by FUDCo**
Partially addresses #7092

- Syscall stats are now aggregated with individual syscall types represented via a dimension rather than as individual stat counters.

- V8 heapStats and kernel process memory usage are now provided.

Other items from #7092 are not addressed here:

- Breaking out various vat-derived data on a per vat basis isn't feasible in the current SwingSet implementation because the data is not captured in a form that lends itself to interpretation in this way.  Longer term we can do this but it will require a fairly extensive overhaul of the kernel's metrics collection machinery (which we might want to do in conjunction with a broader metrics collection overhaul that aims to capture data, such as XS heap usage, from the various individual vat worker processes).

- The requested `queueAllowed_inbound` numbers are not available to the kernel or to cosmic-swingset.  It appears these are in the domain of the Go layer.  They can probably be provided by having the Go layer support its own Prometheus feed, which, if we decide it's worth it, probably should be broken out into its own ticket that can be handed off to one of the folks who maintain the Go layer.

### CODE REVIEW

**Comment by michaelfig** [comment]
> provided by having the Go layer support its own Prometheus feed

FWIW, the Go side already exports several Prometheus metrics to its own endpoint, accessible to Cosmos SDK-based servers like our `agoric-sdk/golang/cosmos` via the `github.com/cosmos/cosmos-sdk/telemetry` package.  So this route may be straightforward.

File: packages/cosmic-swingset/src/kernel-stats.js [hunk 1]
```diff
@@ -361,6 +381,46 @@ export function exportKernelStats({
     }
   }
 
+  let heapStatsLast = 0;
```
Comment (michaelfig) [inline packages/cosmic-swingset/src/kernel-stats.js:387]: I thought there'd be something easier.  Maybe document the fact that I tried...

```suggestion
  // TODO: We probably should not roll our own Node.js process metrics, but a cursory search for
  // "opentelemetry node.js v8 instrumentation" didn't reveal anything really useful.
  let heapStatsLast = 0;
```

**Review by michaelfig — APPROVED** [review]
Have you tried running the chain with metrics enabled and seeing the results?

If you have, full speed ahead!

```
cd packages/cosmic-swingset
# ensure there aren't any weird errors being printed
make scenario2-setup scenario2-run-chain
# see what the metrics look like (once the first block is produced)
curl http://localhost:9464/metrics
```

**Comment by FUDCo** [comment]
> Have you tried running the chain with metrics enabled and seeing the results?

Yes.  @arirubinstein coached me through setting it up.

**Pull request merged**